OSDOCS-6035: Network Observability w/o Loki (#62901)
openshift/openshift-docs: modules/network-observability-without-loki.adoc

// module included in the following assemblies:
// networking/network_observability/installing-operators.adoc

> :_content-type: CONCEPT

Reviewer: Maybe more of a `REFERENCE`? Probably could go either way.

[id="network-observability-without-loki{context}"]
= Network Observability without Loki

> You can use Network Observability without Loki by omitting the Loki installation steps and only installing Kafka. The following table compares available features with and without Loki:

Reviewer 2: ```suggestion
You can use Network Observability without Loki by omitting the Loki installation steps and only exporters. The following table compares available features with and without Loki:
```

Exporters can be Kafka / IPFIX and more in future releases :smile_cat:

Author: Ah, good clarification. TY!

> You can use Network Observability without Loki by omitting the Loki installation steps and only using exporters, such as Kafka or IPFIX. The following table compares available features with and without Loki:

Reviewer 2: I would say yes, IPFIX epic is [release pending](https://issues.redhat.com/browse/NETOBSERV-962) but without target version. QE is also done :heavy_check_mark: 
@skrthomas [was it planned on your side](https://issues.redhat.com/browse/OSDOCS-6348) ?

> You can use Network Observability without Loki by omitting the Loki installation steps and only using exporters, such as Kafka or IPFIX. The following table compares available features with and without Loki:

Author: @jpinsonneau @Amoghrd yep, it is planned, but no in-progress PR yet...coming though!

> You can use Network Observability without Loki by omitting the Loki installation steps and only using exporters, such as Kafka or IPFIX. The following table compares available features with and without Loki:

Reviewer 3: no worries, just wanted to confirm

> You can use Network Observability without Loki by omitting the Loki installation steps and only using exporters, such as Kafka or IPFIX. The following table compares available features with and without Loki:

Author: FYI, IPFIX PR: https://github.com/openshift/openshift-docs/pull/63324

> You can use Network Observability without Loki by omitting the Loki installation steps and only using exporters, such as Kafka or IPFIX. The following table compares available features with and without Loki:

Reviewer 4: "Only" is a tricky word to place. Maybe after the verb is less restrcitive?

> and by using only exporters to X

Reviewer 5: s/by omitting the Loki installation steps/not performing the Loki installation steps in the following section

> .Comparison of feature availability with and without Loki

Reviewer 4: Nice!

Reviewer 4: `X` might be self-explanatory, but do we need to state:

> , where `X` indicates Network Oberservebility supports a Loki or non-Loki capability.

Reviewer 5: s/by omitting the Loki installation steps/not performing the Loki installation steps in the following section

Reviewer 5: I find `x` self-explanatory, especially in context with N/A. However, a check mark might be better

Author: I'm just going to leave the X. I think its self-explanatory and I also don't feel like making an icon for this

Author: I revisit this suggestion and added checks and x's because I have come to think this is a worthwhile and clearer presentation. Thanks for putting the idea in my brain :)

[options="header"]
|============================================================================================================================
|                                     | *With Loki* | *Without Loki* 
| *Exporters*                         | X           | X      
| *Flow-based dashboards*             | X           | X       

> | *Traffic Flow and Topology views*   | X           | N/A     

Reviewer 2: ```suggestion
| *Traffic Flow Overview, Table and Topology views*   | X           | N/A     
```

| *Quick Filters*                     | X           | N/A    

> | *Web console integration*           | X           | N/A

Reviewer 2: ```suggestion
| *OCP console Network Traffic tab integration*           | X           | N/A
```

What about a more explicit description here ?

Author: Good w/ me

Author: @jpinsonneau I guess this row is basically the same as the *Traffic Flow Overview, Table and Topology views* row. Do you have a preference for how we describe this? Or do you think both are necessary?

Reviewer 2: These 2 are parts of the console plugin but are actually different:
- Overview / Table / Topology are `views` we provide with the plugin
- Network Traffic tab integration are `tab entry` added to kubernetes objects (Pod / Service / DeamonSet ...) pages and bringing the user to our `views` with proper filtering

> |============================================================================================================================

Reviewer 3: Thinking out loud, would it be good to add a link on how to set up the exporters here so that it would be easier for a customer to set it up?

Author: Good idea. I can add it to the assembly file as an additional resource situated directly under this.

Author: This link is already an additional resource at the end of the page, but I'm going to move it up to this topic instead.
